app/views/alaveteli_pro/subscriptions/_subscription.html.erb

```diff
@@ -1,41 +1,66 @@
-<%= subscription.plan.name %>
 
-<br>
+<div class="plan-overview">
+  <div class="plan-overview__desc">
+    <%= subscription.plan.name %>
+  </div>
+  <div class="plan-overview__details">
 
-<%= _('Amount: {{plan_price}}',
-      plan_price: number_to_currency(subscription_amount(subscription) / 100.0,
-                                     unit: '£')) %>
+  </div>
+  <label class="plan-overview__amount amount">
+    <span>
+      <%= _('{{plan_price}}',
+            plan_price: number_to_currency(subscription_amount(subscription) / 100.0,
+                                           unit: '£')) %>
+      <span class="amount__freq"><%= billing_frequency(subscription.plan.interval) %>
+      <% if subscription.discounted? %>
+        <%= _('(with <b>{{discount_name}}</b> discount applied)',
+              :discount_name => subscription.discount.coupon.id) %>
+      <% end %>
+      </span>
+    </span>
+  </label>
+</div>
 
-<% if subscription.discounted? %>
-  <%= _('(with <b>{{discount_name}}</b> discount applied)',
-        :discount_name => subscription.discount.coupon.id) %>
-<% end %>
+<div>
+  <h4><%= _('Billing') %></h4>
+  <table class="settings__table">
+    <tr>
+      <th scope="row"><%= _('Subscription began') %></th>
+      <td><%= _('{{date}}',
+            date: simple_date(Time.zone.at(subscription.created),
+                              :format => :text)) %>
+      </td>
+    </tr>
 
-<br>
+    <% if subscription.cancel_at_period_end %>
+    <tr>
+      <th scope="row"><%= _('Subscription ends') %></th>
+      <td><%= _('{{date}}',
+            date: simple_date(Time.zone.at(subscription.current_period_end),
+                              :format => :text)) %>
+      </td>
+    </tr>
 
-<%= billing_frequency(subscription.plan.interval) %>
+    <% else %>
+    <tr>
+      <th scope="row"><%= _('Next payment') %></th>
+      <td><%= _('{{date}}',
+            date: simple_date(Time.zone.at(subscription.current_period_end),
+                              :format => :text)) %>
+      </td>
+    </tr>
 
-<br>
+    <% end %>
+  </table>
 
-<%= _('Subscription created on {{date}}',
-      date: simple_date(Time.zone.at(subscription.created),
-                        :format => :text)) %>
-
-<br>
-
-<% if subscription.cancel_at_period_end %>
-  <%= _('Your subscription will end on {{date}}',
-        date: simple_date(Time.zone.at(subscription.current_period_end),
-                          :format => :text)) %>
-<% else %>
-  <%= _('Your next payment will be taken on {{date}}',
-        date: simple_date(Time.zone.at(subscription.current_period_end),
-                          :format => :text)) %>
-
-  <br>
+</div>
 
+<div class="cancel-sub-section">
```

Comment:
This still displays the "cancel" link even if the subscription is cancelled. There was previously a conditional

![screen shot 2017-10-12 at 14 52 52](https://user-images.githubusercontent.com/282788/31500338-0cff2e46-af5f-11e7-9887-caa8d778c870.png)